src/app/shared/table/table.component.html

```diff
@@ -92,6 +92,21 @@
         <ng-template ngFor let-filterDef [ngForOf]="dataSource.tableFiltersDef">
           <div *ngIf="filterDef.visible !== false"
             [class]="(filterDef.class ? filterDef.class : '')">
+            <!-- Button based Dropdown filter -->
+            <div *ngIf="filterDef.type === FilterType.MENU_DROPDOWN">
+              <button mat-stroked-button [matMenuTriggerFor]="menu">
```

Comment:
If it's possible also, reduce the width of the button so we have a square button and not a rectangular one.